ansible/README.md

```diff
@@ -1,33 +1,108 @@
 OME Ansible
 ===========
 
-This contains a variety of Ansible playbooks and roles, including example files for provisioning an OpenStack VM from scratch with OMERO using Ansible.
+This folder contains a variety of Ansible playbooks, including example files for provisioning an OpenStack VM from scratch with OMERO using Ansible.
 Most of these scripts should also work on other platforms, providing the VM is brought up by some other method.
 
 - Playbooks which start with `os` are OpenStack specific - the `os` stands for `OpenStack`.
 - Playbooks which start with `idr` are for the Image Data Repository.
 - Playbooks which start with `ci` are for OME continuous integration and build.
 
-To get started with a minimal OMERO server, use the example playbook in the [README.md of roles/omero-server](/roles/omero-server/README.md).
-
+To get started with a minimal OMERO server, use the example playbook in the [omero-server](https://github.com/openmicroscopy/ansible-role-omero-server) role.
 
 Roles
 -----
-There are two roles directories.
-- `roles`: These are roles which are considered ready for use. Breaking changes to these roles will be minimised.
-- `roles-dev`: Roles which are still in development, or require special external configuration. These are not recommended for use.
+
+Roles for these playbooks can be found on http://galaxy.ansible.com or http://github.com .
+A list of most of the roles can be found in
+[ansible/requirements.yml](https://github.com/openmicroscopy/infrastructure/blob/master/ansible/requirements.yml).
+
+### Partial list of core roles ###
```

Comment:
How do you view a "core" role? One that's core to our internal infrastructure? Or one that should be highlighted to the external community?